### ISSUE TITLE
"Simultaneous subscription" error with Argus PT

### ISSUE THREAD

**Issue by jgbaum** [QuantumEntangledAndy/neolink#128]
**Describe the bug**
Error message when trying to connect to [Argus PT](https://reolink.com/product/argus-pt/): failed to connect to camera, simultaneous subscription

Here's the error message, with the UID redacted:

```
[2023-08-06T20:20:48Z INFO  neolink_core::bc_protocol] cam96: Trying local discovery
[2023-08-06T20:20:51Z INFO  neolink_core::bc_protocol] cam96: Trying relay discovery
[2023-08-06T20:20:51Z INFO  neolink_core::bc_protocol] cam96: Trying remote discovery
[2023-08-06T20:20:51Z INFO  neolink_core::bc_protocol] cam96: Trying map discovery
[2023-08-06T20:20:51Z WARN  neolink::rtsp] cam96: Retryable error: cam96: Could not connect to camera
    
    Caused by:
        0: Failed to connect to camera cam96 at UID: {REDACTED} on channel 0
        1: Simultaneous subscription, Some(0)
[2023-08-06T20:20:52Z INFO  neolink_core::bc_protocol] cam96: Trying local discovery
[2023-08-06T20:20:53Z INFO  neolink_core::bc_protocol] cam96: Trying map discovery
[2023-08-06T20:20:53Z INFO  neolink_core::bc_protocol] cam96: Trying relay discovery
[2023-08-06T20:20:53Z INFO  neolink_core::bc_protocol] cam96: Trying remote discovery
[2023-08-06T20:20:53Z WARN  neolink::rtsp] cam96: Retryable error: cam96: Could not connect to camera
    
    Caused by:
        0: Failed to connect to camera cam96 at UID: {REDACTED} on channel 0
        1: Simultaneous subscription, Some(0)
[2023-08-06T20:20:55Z INFO  neolink_core::bc_protocol] cam96: Trying local discovery
[2023-08-06T20:20:55Z INFO  neolink_core::bc_protocol] cam96: Trying relay discovery
[2023-08-06T20:20:55Z INFO  neolink_core::bc_protocol] cam96: Trying remote discovery
[2023-08-06T20:20:55Z INFO  neolink_core::bc_protocol] cam96: Trying map discovery
[2023-08-06T20:20:55Z WARN  neolink::rtsp] cam96: Retryable error: cam96: Could not connect to camera
    
    Caused by:
        0: Failed to connect to camera cam96 at UID: {REDACTED} on channel 0
        1: Simultaneous subscription, Some(0)
[2023-08-06T20:21:00Z INFO  neolink_core::bc_protocol] cam96: Trying local discovery
[2023-08-06T20:21:02Z INFO  neolink_core::bc_protocol] cam96: Trying relay discovery
[2023-08-06T20:21:02Z INFO  neolink_core::bc_protocol] cam96: Trying remote discovery
[2023-08-06T20:21:02Z INFO  neolink_core::bc_protocol] cam96: Trying map discovery
[2023-08-06T20:21:02Z WARN  neolink::rtsp] cam96: Retryable error: cam96: Could not connect to camera
    
    Caused by:
        0: Failed to connect to camera cam96 at UID: {REDACTED} on channel 0
        1: Simultaneous subscription, Some(0)
Good night
```

This is the docker image that I'm using: [quantumentangledandy/neolink](https://hub.docker.com/r/quantumentangledandy/neolink)

Relevant config portion with sensitive data redacted:
```
name = "cam96"
username = "admin"
password = "{REDACTED}"
uid = "{REDACTED}"
stream = "mainStream"
```

Previously, I tried [this docker image](https://hub.docker.com/r/thirtythreeforty/neolink) but ran into an issue similar to https://github.com/thirtythreeforty/neolink/issues/357

Please let me know if there's any additional info you need to assist.  Thanks!

P.S. I'm psyched that I stumbled onto this project and hope I can make use of it!

**Comment by QuantumEntangledAndy** [comment]
Yes It's one of the issues already resolved by #127 a long with the new firmware changes. Should be merged later today.

**Comment by jgbaum** [comment]
Awesome, looking forward to it!

**Comment by QuantumEntangledAndy** [comment]
Test build is [here](https://github.com/QuantumEntangledAndy/neolink/actions/runs/5780812549) if you can please check it works for you

**Comment by QuantumEntangledAndy** [comment]
Should be now fixed in 0.5.17

**Comment by jgbaum** [comment]
> Should be now fixed in 0.5.17

Yes, thank you!
